client/mass/store.js

```diff
@@ -97,7 +97,7 @@ class TdbStore {
 				let plot
 				try {
 					const _ = await import(`../plots/${savedPlot.chartType}.js`)
-					plot = await _.getPlotConfig(savedPlot, this.app)
+					plot = await _.getPlotConfig(savedPlot, this.app, this.state)
```

Comment:
this is not a good idea, `this.state` is not fully computed until after state rehydration. So getPlotConfig() when called as part of state rehydration should not expect a full state.

Reply:
Ok, how can I get the active cohort in this case?